Makefile

```diff
@@ -483,7 +492,7 @@ endif
 
 ifeq ($(TARGET_PLATFORM),win32)
 electron-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
-electron-installer-nsis: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
+electron-installer-nsis: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
```

Comment:
Oh, I missed this comment. The snapshot build publishing step still works correctly, since it will upload the file pointed by the new path. I'll move it to `dist/out` anyway!